index.bs

```diff
@@ -5796,6 +5800,41 @@ At this time, one [=credential property=] is defined: the [=resident key credent
 
 For the purposes of user agent automation and [=web application=] testing, this document defines a number of [[WebDriver]] [=extension commands=].
 
+## WebAuthn WebDriver Capability ## {#sctn-automation-webdriver-capability}
+
+In order to advertise the availability of the [=extension commands=] defined below, a new [=extension capability=] is added.
+
+<figure id="table-virtualAuthenticatorsWebdriverCapability" class="table">
+    <table class="data">
+        <thead>
+            <tr>
+                <th>Capability</th>
+                <th>Key</th>
+                <th>Value Type</th>
+                <th>Description</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>Virtual Authenticators Support</td>
+                <td>`":virtualAuthenticators"`</td>
+                <td>boolean</td>
+                <td>Indicates whether the [=endpoint node=] supports all [=Virtual Authenticators=] commands.</td>
+            </tr>
+        </tbody>
+    </table>
+</figure>
+
+When [=validating capabilities=], the extension-specific substeps to validate `":webauthn"` with `value` are the following:
```

Comment:
should this rather be `:virtualAuthenticators` ?

Reply:
Yes, done.